### ISSUE TITLE
ktrain import fails due to failure to import master_bar from fastprogress 

### ISSUE THREAD

**Issue by AdamBrownLab** [amaiya/ktrain#40]
I installed ktrain on my GPU server (running ubuntu 18.04.2)
Trying to import ktrain failed:

```
>>> import ktrain
using Keras version: 2.2.4-tf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/ktrain/__init__.py", line 2, in <module>
    from .imports import *
  File "/usr/local/lib/python3.6/dist-packages/ktrain/imports.py", line 173, in <module>
    from fastprogress import master_bar, progress_bar
ImportError: cannot import name 'master_bar'
```

Reinstalling fastprogress did not solve the issue; indeed, `import  fastprogress `does not fail

**Comment by amaiya** [comment]
I was able to reproduce this issue.  This appears to be some sort of bug in the latest version of `fastprogress`, which was just released a couple of days ago.  The resolution is to downgrade `fastprogress` to v0.1.22:

```
pip3 install fastprogress==0.1.22
```
After doing this, you should be able to import *ktrain* with no problem.

Thanks.

**Comment by marlingod** [comment]
I change the:
from fastprogress import master_bar, progress_bar
to
from fastprogress.fastprogress import master_bar, progress_bar
in the ~/fastai/imports/core.py file and it worked

**Comment by amaiya** [comment]
Thanks - this is was fixed in v0.7.3,  so that both old-style and new-style `fastprogress` imports are supported.
